### PR TITLE
Translation error #1358 - fixed French mistranslation

### PR DESCRIPTION
**Pull request by AdamFx990**

### CODE REVIEW

File: CorsixTH/Lua/languages/french.lua [hunk 1]
```diff
@@ -207,7 +207,7 @@ adviser.information = {
   patient_leaving_too_expensive = "Un patient part sans payer la facture pour %s. Sacrée perte !",
   vip_arrived = "Attention ! %s arrive pour visiter votre hôpital ! Faites en sorte de lui faire bonne impression.",
   first_death = "Vous venez de tuer votre premier patient. Alors, heureux ?",
-  promotion_to_specialist = "L'un de vos INTERNES est devenu MÉDECIN.", -- Fix the famous "Level 5 bug"
```
Comment (wolfy1339) [inline CorsixTH/Lua/languages/french.lua]: Missing an `E` at the end there

Comment (TheCycoONE) [reply CorsixTH/Lua/languages/french.lua]: Was going to ask your opinion

Comment (AdamFx990) [reply CorsixTH/Lua/languages/french.lua]: So you reckon it should be "MÉDECINE"?

File: CorsixTH/Lua/languages/french.lua [hunk 2]
```diff
@@ -207,7 +207,7 @@ adviser.information = {
   patient_leaving_too_expensive = "Un patient part sans payer la facture pour %s. Sacrée perte !",
   vip_arrived = "Attention ! %s arrive pour visiter votre hôpital ! Faites en sorte de lui faire bonne impression.",
   first_death = "Vous venez de tuer votre premier patient. Alors, heureux ?",
-  promotion_to_specialist = "L'un de vos MÉDECIN est devenu un SPÉCIALIST.", -- Fix the famous "Level 5 bug"
+  promotion_to_specialist = "L'un de vos MÉDECINE est devenu un SPÉCIALIST.", -- Fix the famous "Level 5 bug"
```
Comment (wolfy1339) [inline CorsixTH/Lua/languages/french.lua]: not `MÉDECINE` but `SPÉCIALISTE`.

**Pull request merged**